tests/playwright/config/global-setup.js

```diff
@@ -6,11 +6,11 @@ module.exports = async ( config ) => {
 	const browser = await chromium.launch( { headless: config.headless } ),
 		page = await browser.newPage();
 
-	await page.goto( `${ config.baseURL }/wp-admin` );
+	await page.goto( `${ config.baseURL }/wp-admin`, { waitUntil: 'networkidle' } );
```

Comment:
Is it possible to undo your changes in this file? Because this will affect all users.

Reply:
We discussed with @Nevoss that this would be a good solution. 
Often when running the tests locally it fails to log into Wordpress. So I change it every time locally to consistently have the tests run.

Reply:
I tried to replace 'networkidle' with 'load' and the tests pass the authentication on WP every other time.